README.md

```diff
@@ -654,198 +659,199 @@ Now we will create our `Initialize` method step by step:
 The stack will then be loaded into the execution segment in the next method. For now, the stack will be empty, but later on it will contain the builtin bases (which are the arguments for the main function), and the function arguments when running a function from a starknet contract.
 
 ```go
 func (r *CairoRunner) initializeFunctionEntrypoint(entrypoint uint, stack *[]memory.MaybeRelocatable, return_fp memory.Relocatable) (memory.Relocatable, error) {
-	end := r.Vm.Segments.AddSegment()
-	*stack = append(*stack, *memory.NewMaybeRelocatableRelocatable(end), *memory.NewMaybeRelocatableRelocatable(return_fp))
-	r.initialFp = r.executionBase
-	r.initialFp.Offset += uint(len(*stack))
-	r.initialAp = r.initialFp
-	r.finalPc = end
-	return end, r.initializeState(entrypoint, stack)
+    end := r.Vm.Segments.AddSegment()
+    *stack = append(*stack, *memory.NewMaybeRelocatableRelocatable(end), *memory.NewMaybeRelocatableRelocatable(return_fp))
+    r.initialFp = r.executionBase
+    r.initialFp.Offset += uint(len(*stack))
+    r.initialAp = r.initialFp
+    r.finalPc = end
+    return end, r.initializeState(entrypoint, stack)
 }
 ```
 
-*InitializeState*
+##### InitializeState
 
 This method will be in charge of loading the program data into the program segment and the stack into the execution segment
 
 ```go
 func (r *CairoRunner) initializeState(entrypoint uint, stack *[]memory.MaybeRelocatable) error {
-	r.initialPc = r.ProgramBase
-	r.initialPc.Offset += entrypoint
-	// Load program data
-	_, err := r.Vm.Segments.LoadData(r.ProgramBase, &r.Program.Data)
-	if err == nil {
-		_, err = r.Vm.Segments.LoadData(r.executionBase, stack)
-	}
-	return err
+    r.initialPc = r.ProgramBase
+    r.initialPc.Offset += entrypoint
+    // Load program data
+    _, err := r.Vm.Segments.LoadData(r.ProgramBase, &r.Program.Data)
+    if err == nil {
+        _, err = r.Vm.Segments.LoadData(r.executionBase, stack)
+    }
+    return err
 }
 ```
 
-*initializeVm*
+##### initializeVm (CairoRunner)
 
 This method will set the values of the VM's `RunContext` with our `CairoRunner`'s initial values
 
 ```go
 func (r *CairoRunner) initializeVM() {
-	r.Vm.RunContext.Ap = r.initialAp
-	r.Vm.RunContext.Fp = r.initialFp
-	r.Vm.RunContext.Pc = r.initialPc
+    r.Vm.RunContext.Ap = r.initialAp
+    r.Vm.RunContext.Fp = r.initialFp
+    r.Vm.RunContext.Pc = r.initialPc
 }
 ```
 
 With `CairoRunner.Initialize()` now complete we can move on to the execution step:
 
-*RunUntilPc*
+##### RunUntilPc
 
 This method will continuously execute cairo steps until the end pc, returned by 'CairoRunner.Initialize()' is reached
 
 ```go
     //TODO
 ```
 
-*Step*
+##### Step
 
 ```go
     //TODO
 ```
 
-*Decode instruction*
+##### Decode instruction
 
 ```go
     //TODO
 ```
 
-*Run instruction*
+##### Run instruction
 
 ```go
     //TODO
 ```
 
-*Compute operands*
+##### Compute operands
 
 ```go
     //TODO
 ```
 
-*Opcode assertions*
+##### Opcode assertions
```

Comment:
We should move these sections to the `Instruction Decoding and Execution` section (line 493)

Reply:
done

Reply:
My bad! It was line 552 (line 493 is before the Write your own vm section, in the general overview section)

Reply:
done!